interop-tests/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 edition = "2021"
+license = "MIT"
 name = "interop-tests"
-version = "0.1.0"
 publish = false
-license = "MIT"
+version = "0.1.0"
```

Comment:
I am fine with these changes. Though to reduce the noise of this already **very** large pull request, please do them in a separate pull request.

Reply:
Sorry about that! My TOML formatter puts everything alphabetical as Thomas suggested. I can revert these changes